### PR TITLE
Add a condition to handle cases when act['x-window'] is undefined.

### PR DESCRIPTION
**Pull request by Damus765**
see : https://github.com/deadalnix/pixel-saver/issues/14

### CODE REVIEW

File: pixel-saver@deadalnix.me/decoration.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ function guessWindowXID(win) {
 	
 	// use xwininfo, take first child.
 	let act = win.get_compositor_private();
-	if (act) {
+	if (act && act['x-window']) {
```
Comment (deadalnix) [inline pixel-saver@deadalnix.me/decoration.js]: let xwindow = act && act['x-window'];
if (xwindow) { ...

File: pixel-saver@deadalnix.me/decoration.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ function guessWindowXID(win) {
...
 		let xwininfo = GLib.spawn_command_line_sync('xwininfo -children -id 0x%x'.format(act['x-window']));
```
Comment (deadalnix) [inline pixel-saver@deadalnix.me/decoration.js]: Resuse xinwidow here.

**Comment by deadalnix** [comment]
I'm not sure why you deleted the PR and recreated it. You loose the comment from the previous one in the process.

**Comment by Damus765** [comment]
Yeah, I'm pretty new to git, and I thought the PR needed to be recreated. My bad. However, I've seen your comment on the previous PR. I'm just wondering, do we need to test act && act['x-window'] ? If act['x-window'] is defined and not null, act should also be so. So does this do the trick ?
let xwindow = act['x-window'];
if (xwindow) { ...

**Comment by deadalnix** [comment]
I'm not sure what the semantic of falsy['string']. I'd rather not try to get into the ill defined parts of the language.
